_maps/map_files/BoxStation/BoxStation.dmm

```diff
@@ -33655,6 +33655,11 @@
 	dir = 8
 	},
 /obj/machinery/atmospherics/pipe/simple/scrubbers/hidden,
+/obj/machinery/power/proto_sh_emitter/small/anchored{
+	icon_state = "proto_sh_emitter";
```

Comment:
dv?

Reply:
Uhh what?

Reply:
You shouldn't have to set these icons manually on the maps since it's their normal icon.

Reply:
uh i didn't, the map thing did it, i can webedit it tho